README.md

```diff
@@ -288,9 +288,12 @@ ${hostedzone://example.com/private}
 
 Looks up an AMI given a name prefix which may contain wildcards. If query returns multiple images the latest is used.
 
+Owners takes a query string list of AWS account ID, self (owner is the sender of the request), or an AWS owner alias (valid values are amazon | aws-marketplace | microsoft). Omitting this option returns all images for which you have launch permissions, regardless of ownership.
```

Comment:
I copied that description from here .. https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html#describe_images-instance_method for options: owners